jobs/ocp4/pds-ocp4-install.yaml

```diff
@@ -0,0 +1,258 @@
+                                '''
+                            )
+                        }
+
+                        sh(
+                            returnStdout: false,
+                            script: """
+                                sed -i "s@http://master.example.com@https://console-openshift-console.apps.cluster-${RHPDS_CITY}.${RHPDS_CITY}.openshiftworkshop.com@" ./integreatly-operator/deploy/crds/examples/installation.cr.yaml
+                                sed -i "s@example.com@apps.cluster-${RHPDS_CITY}.${RHPDS_CITY}.openshiftworkshop.com@" ./integreatly-operator/deploy/crds/examples/installation.cr.yaml
+                                sed -i "s@integreatly-@${namespacePrefix}-integreatly-@" ./integreatly-operator/deploy/crds/examples/installation.cr.yaml
+                                sed -i "s@type: workshop@type: ${INSTALLATION_TYPE}@" ./integreatly-operator/deploy/crds/examples/installation.cr.yaml
+
+                                until ./oc get deployments integreatly-operator -n ${namespaceName} --output=json -o jsonpath='{.status.availableReplicas}' | grep -m 1 "1"; do echo "Operator not ready yet. Sleeping for 5 seconds";sleep 5 ; done
+                                ./oc create -f ./integreatly-operator/deploy/crds/examples/installation.cr.yaml -n ${namespaceName}
+                            """
+                        )
+                    } // stage
+
+                    stage ('Verify Installation Has Finished') {
+                        sh(
+                            returnStdout: false,
+                            script: """
+                                until ./oc get Installation example-installation -n ${namespaceName} --output=json -o jsonpath='{.status.stages.authentication.phase}' | grep -m 1 "completed"; do echo "Authentication stage not completed yet. Sleeping for 20 seconds";sleep 20 ; done
+
+                                until ./oc get Installation example-installation -n ${namespaceName} --output=json -o jsonpath='{.status.stages.products.phase}' | grep -m 1 "completed"; do echo "Products stage not completed yet. Sleeping for 20 seconds";sleep 20 ; done
+                            """
+                        )
+                    } // stage
+                } catch (caughtError){
+                    currentBuild.result = 'FAILURE'
+                    err = caughtError
+                    println caughtError
+                } finally {
+                    notifyBuild(currentBuild.result, err)
+                    
+                    if (cleanupAfterward) {
+                        cleanup(clusterAPI, namespaceName)
+                    }
+                }
+            }}} // timeout, ansiColor, timestamps
+        } // node
+
+        def notifyBuild(String buildStatus, err) {
+
+            // In theory, null means success. In practise if there is an error thrown elsewhere than in the triggered jobs (eg. in wokraround scripts)
```

Comment:
small typo here: `wokraround`

Reply:
nice catch! I copied this so we have this typo in more places. will create additional PR when I fix all the occurrences at once.